api/v1alpha1/nificluster_types.go

```diff
@@ -374,7 +372,6 @@ type SSLSecrets struct {
 	// https://cert-manager.io/docs/concepts/issuer/
 	IssuerRef *cmmeta.ObjectReference `json:"issuerRef,omitempty"`
 	// TODO : add vault
-	// +kubebuilder:validation:Enum={"cert-manager","vault"}
```

Comment:
Why this change ?

Reply:
To make this kubebuilder annotation available to all the resources using the PKIBackend type to avoid having to copy and paste.